src/controllers/userController.js

```diff
@@ -1,29 +1,33 @@
-const userService = require('../services/userService');
-const logger = require('../helpers/logger');
+const userService = require('../services/userService')
+const logger = require('../helpers/logger')
 
 const createUser = async (req, res) => {
```

Comment:
don't you think that this endpoint doesn't create user, it just subscribes user`s email for daily notification? So maybe it is better to name this function something like subscribeUser or subscribeEmail to improve readability